packages/babel-plugin-minify-simplify/__tests__/simplify-test.js

```diff
@@ -1797,6 +1797,31 @@ describe("simplify-plugin", () => {
     expect(transform(source)).toBe(expected);
   });
 
+  it("should ignore statement after return", () => {
+    const source = unpad(`
+      function bar(foo) {
+        switch (foo) {
+          case 'foo':
+            console.log('foo')
+            return 1;
+            console.log('bar')
+          case 'bar':
+            return 2;
+          default:
+            return 3;
+        }
+      }
+    `);
+
+    const expected = unpad(`
+      function bar(foo) {
+        return foo === 'foo' ? (console.log('foo'), 1) : foo === 'bar' ? 2 : 3;
```

Comment:
As a usual behavior, it don't simplify anything. Should I add it as a test case?